src/test/java/org/jabref/logic/importer/fetcher/WorldcatFetcherTest.java

```diff
@@ -0,0 +1,52 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.util.List;
+
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.testutils.category.FetcherTest;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@FetcherTest
+public class WorldcatFetcherTest {
+
+    private WorldcatFetcher fetcher;
+
+    @BeforeEach
+    public void setUp() {
+        fetcher = new WorldcatFetcher("API_KEY");
```

Comment:
Can you please use the key `aMHOf2rfzUt3fuKkb7DXX8pkBv1AmcBWwwoSfwpt8CMhdUdxXscB4ESOmBPs4NlmYJmFtcSZ3Q5kMxzb`. This is a sandbox key that should work (for the build server and for local testing)

Reply:
Missed to resolve this, but this is the token being used